pkg/services/ngalert/remote/alertmanager.go

```diff
@@ -466,6 +466,9 @@ func (am *Alertmanager) GetReceivers(ctx context.Context) ([]apimodels.Receiver,
 
 	var rcvs []apimodels.Receiver
 	for _, rcv := range res.Payload {
+		if rcv.Integrations == nil {
```

Comment:
The front-end expects the `integrations` field to be an array, having `null` here causes an error.